packages/office-ui-fabric-react/src/components/DetailsList/ShimmeredDetailsList.types.ts

```diff
@@ -1,24 +1,83 @@
 import { IDetailsListProps } from './DetailsList.types';
+import { IDetailsRowProps } from './DetailsRow.types';
 import { IStyle, ITheme } from '../../Styling';
 import { IStyleFunctionOrObject } from '../../Utilities';
 
+/**
+ * ShimmeredDetailsList props interface
+ * {@docCategory ShimmeredDetailsList}
+ * @deprecated The inheritance from `IDetailsListProps` will be dropped in favor of `detailsListProps` in Fabric 7.0.
+ */
 export interface IShimmeredDetailsListProps extends IDetailsListProps {
-  /** The theme provided by context */
-  theme?: ITheme;
+  /**
```

Comment:
This is confusing because `IShimmeredDetailsListProps extends IDetailsListProps`. I think what you might want to do is take a function which remaps them or something:
```ts
onGetDetailsListProps?(detailsListProps: IDetailsListProps): IDetailsListProps;
```
Basically, this says "here is what ShimmeredDetailsList was going to do. Do you want to override anything before `DetailsList` itself is rendered?"